### ISSUE TITLE
`Layout/MultilineMethodCallIndentation` - possible false positive

### ISSUE THREAD

**Issue by mlwyatt** [rubocop/rubocop#14743]
This may be by design or it could be a false positive for `Layout/MultilineMethodCallIndentation`. If this is by design, I'd like an option to include the splat (`*`) as part of the receiver

--------

## Steps to reproduce the problem

```rb
# /Easy/lint_me.rb

# frozen_string_literal: true

_data = [
  *(1..100)
    .map { |i| "really really really really really really really really really really long string that contains #{i}" }
].join("\n")
```

```yml
# /Easy/.rubocop.yml

AllCops:
  NewCops: disable
  Enabled: false
  SuggestExtensions: false

Layout/MultilineMethodCallIndentation:
  Enabled: true
  EnforcedStyle: indented_relative_to_receiver
```

## Expected behavior

No cops register offenses

## Actual behavior

```
rubocop ./lint_me.rb --debug
For /Easy: configuration from /Easy/.rubocop.yml
Default configuration from /usr/local/bundle/gems/rubocop-1.82.0/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /Easy/lint_me.rb
Loading cache from /root/.cache/rubocop_cache/08a201591d3e98ed16a8b6ca6ca46fd57afd17ee/6d7a3b621ca1730e04accd938619e4bdab66cfb1/9d637f2b24251183a97572df93496c144aab42dc
C

Offenses:

lint_me.rb:5:5: C: [Correctable] Layout/MultilineMethodCallIndentation: Indent .map 2 spaces more than (1..100) on line 4.
    .map { |i| "really really really really really really really really really really long string that contains #{i}" }
    ^^^^

1 file inspected, 1 offense detected, 1 offense autocorrectable
Finished in 0.06988658299087547 seconds
```

## RuboCop version

```
$ rubocop -V
1.82.0 (using Parser 3.3.8.0, Prism 1.6.0, rubocop-ast 1.48.0, analyzing as Ruby 3.4, running on ruby 3.4.4) [aarch64-linux]
```
